### PR TITLE
Create 2023-03-20-top-tasks

### PR DESCRIPTION
**Pull request by prycrane**
## Preview links
- fr: https://deploy-preview-126--blogue-sct.netlify.app/2023/03/17/les-taches-principales
- en: https://deploy-preview-126--blog-tbs.netlify.app/2023/03/17/top-tasks.html

### CODE REVIEW

**Comment by prycrane** [comment]
I am having trouble getting the English side to compile.

**Review by delisma — CHANGES_REQUESTED** [review]
Good work! We're just missing the Open Graph images.

File: en/_posts/2023-03-17-top-tasks.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+css:
+- ../../../css/blog.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
```
Comment (delisma) [inline en/_posts/2023-03-17-top-tasks.md:4]: Should we move this in the _config.yml in a different PR? If not could we move the css property after the altLangPage property to keep the Frontmatter in alphabetical order.

File: en/_posts/2023-03-17-top-tasks.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<p>There are thousands of Government of Canada (GC) tasks out there, some simple, and some complex. So how do we prioritize what to improve? We identify the most used tasks, and we call these the “top tasks”.</p>
+<h2>Top tasks form the basis of the Canada.ca design</h2>
+<p>Top task management isn’t new to the Government of Canada. In 2013, we compiled the first government-wide top task list to help design Canada.ca. Even earlier than that, several departments were experimenting with top task management for their own sites.</p>
+<p>Since then, we’ve gone through 4 major iterations of the top task list. Each time we do it, we improve how we gather comparable data and tasks.</p>
+<h2>How we compiled the previous list</h2>
+<p>In 2017, we asked institutions to submit their lists of top tasks by considering web visits, call centre data, previous surveys and research, mandate letters, and departmental priorities.  When we published the top 100 tasks in 2018, the tasks were simply ranked by visits to the task start pages.</p>
+<p>Using this simple ranking method created inconsistencies in the results such as:</p>
+<ul>
+  <li><strong>Analytics comparability</strong>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply en/_posts/2023-03-17-top-tasks.md:24]: Both in this case, hence the strong tag.

File: en/_posts/2023-03-17-top-tasks.md (hunk 1 above)
Comment (prycrane) [reply en/_posts/2023-03-17-top-tasks.md:4]: I just copied that path from the previous blog.  Not good style, but got the job done.  Yeah, we should fix that, fore sure.

File: fr/_posts/2023-03-17-les-taches-principales.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+css:
+-  ../../../css/blogue.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+altLangPage: /2023/03/17/top-tasks
+date: 2023-03-17
+dateModified: 2023-03-17
+description: "Les tâches principales permettent d’établir l’ordre de priorité du contenu Web qui doit être amélioré. Depuis 2022, les tâches principales pour Canada.ca sont définies à l’aide des réponses fournies par les visiteurs dans le sondage sur la réussite des tâches du gouvernement du Canada."
+title: "Nous changeons la façon dont nous déterminons les tâches principales pour le site Canada.ca"
```
[the remark this answers is not in the document]
Comment (prycrane) [reply fr/_posts/2023-03-17-les-taches-principales.md:9]: Agreed.  Can we leave this for publication?  I will talk to Laura about fixing it.  Alas.  One of those forever problems.  Alas!

File: fr/_posts/2023-03-17-les-taches-principales.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  <li>Faire une demande d’immigration au Canada par l’Entrée express</li>
+  <li>Soumettre une déclaration d’assurance-emploi</li>
+  <li>Voir les renseignements de mon T4</li>
+  <li>Obtenir une copie de mon avis de cotisation aux fins de l’impôt</li>
+  <li>Étudier au Canada – personnes non canadiennes</li>
+  <li>Vérifier le statut d’une demande d’assurance-emploi</li>
+  <li>Prestation canadienne de la relance économique</li>
+</ol>
+<p class="mrgn-tp-md"><a href="https://www.canada.ca/fr/gouvernement/a-propos/taches-principales-pour-canada-ca.html">Voir les 50 tâches principales pour Canada.ca</a></p>
+<h2>Pourquoi utiliser le Sondage sur la réussite des tâches du GC</h2>
```
Comment (LPipes4) [inline fr/_posts/2023-03-17-les-taches-principales.md:63]:
```suggestion
<h2>Pourquoi se servir du Sondage sur la réussite des tâches du GC</h2>
```

**Review by LPipes4 — CHANGES_REQUESTED** [review]

**Review by delisma — APPROVED** [review]

**Review by delisma — APPROVED** [review]

File: en/_posts/2023-03-22-top-tasks.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+date: 2023-03-22
+dateModified: 2023-03-22
+description: "Top tasks help prioritize what web content needs to be improved. As of 2022, top tasks for Canada.ca are identified using visitors’ responses to the GC Task Success Survey."
+title: "We’re changing the way we identify top tasks for Canada.ca"
+---
+<div class="row">
+  <div class="col-md-8"><img src="/images/thumbs/2023-03-22.png" class="img-responsive" alt="" /></div>
+</div>
+<p class="mrgn-tp-lg">People come to our digital channels to accomplish a very wide range of tasks. We say they’re looking for “information” or “services”, but they don’t make these distinctions. They come to Canada.ca with a goal in mind and a task that they want to complete. If they’re able to accomplish their task, their need has been met. If not, we have work to do.</p>
```
Comment (LPipes4) [inline en/_posts/2023-03-22-top-tasks.md:14]: Can we change “services”, to “services,”    ...the comma should be inside the quote

**Pull request merged**